README.md

```diff
@@ -18,33 +24,52 @@ Take a look at some example apps:
+| Video            | [Tavus](https://docs.pipecat.ai/api-reference/services/video/tavus)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | `pip install pipecat-ai[tavus]`     |
+| Vision & Image   | [Moondream](https://docs.pipecat.ai/api-reference/services/vision/moondream), [fal](https://docs.pipecat.ai/api-reference/services/image-generation/fal)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | `pip install pipecat-ai[moondream]` |
+| Analytics        | [Canonical AI](https://docs.pipecat.ai/api-reference/services/analytics/canonical)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | `pip install pipecat-ai[canonical]` |
```

Comment:
We also have Sentry https://github.com/pipecat-ai/pipecat/blob/main/src/pipecat/processors/metrics/sentry.py. This could be called Analytics & Metrics.